app/helpers/table_helper.rb

```diff
@@ -4,4 +4,8 @@ def app_table(*args, **kwargs, &block)
       govuk_table(*args, **kwargs, &block)
     end
   end
+
+  def make_not_yet_known_secondary_colour(placement)
```

Comment:
Nice work, this title is descriptive! A possible change to reduce the cognitive load would be `not_yet_known_colour_class`, we don't want to couple the name to the specific method logic too tightly!